cli/browser-compatibility-table.js

```diff
@@ -0,0 +1,22 @@
+export function fixMdnURL(document) {
+  // Loop over, and mutate, all 'browser_compatibility' sections
+  document.body
+    .filter(section => section.type === "browser_compatibility")
+    .forEach(section => {
+      Object.entries(section.value.data).forEach(([key, block]) => {
+        if (!!block.__compat) {
+          block = block.__compat;
```

Comment:
I think this needs a code comment that explains this. I don't even know why this is needed. I bet in 6 months, you don't know either :)